### ISSUE TITLE
Dataset playback - depth labelling doesn't work with simulation

### ISSUE THREAD

**Issue by danifpdra** [lardemua/atom#420]
FYI @manuelgitgomes @miguelriemoliveira 

![image](https://user-images.githubusercontent.com/47574905/164227521-e730d99a-b6ab-43b2-af55-bc9aa89dcc21.png)

**Comment by miguelriemoliveira** [comment]
Another problem ... 

let me look into it. In the meantime you can work on the real data only.

**Comment by miguelriemoliveira** [comment]
Waiting for https://github.com/lardemua/larcc/issues/45 ...

or you could give me a simulated bagfile...

**Comment by danifpdra** [comment]
A dataset is enought right?

[train_dataset.zip](https://github.com/lardemua/atom/files/8525751/train_dataset.zip) (THIS IS SIMULATION, I HAVE IT UNDER LARCC_SIM IN THE DATASET FOLDER)

**Comment by miguelriemoliveira** [comment]
Hi @danifpdra ,

thanks. I think its enough, but without the bagfile I cannot reconfigure the package.

I just tested this. If I label like this

![image](https://user-images.githubusercontent.com/11350998/164521657-ad838f69-53f5-4a6d-bbc5-379041635cbf.png)

then I get a bad labelling like this:

![image](https://user-images.githubusercontent.com/11350998/164521753-7f5b0c60-3229-462f-af18-edec6a88b013.png)


In this case we have to be careful about defining the limits of the pattern. If I do like this


![image](https://user-images.githubusercontent.com/11350998/164521968-866cbf8b-afd4-4c2d-84fe-1024a373b2c0.png)


We get much better results

![image](https://user-images.githubusercontent.com/11350998/164522022-5523a7d5-4fb1-46c8-8930-08fc828c1819.png)


Here's a video

https://www.youtube.com/watch?v=c0dEl9aPbf8

Is there something to implement here or can we close the issue?

**Comment by danifpdra** [comment]
![image](https://user-images.githubusercontent.com/47574905/164674820-2e87205a-e978-49ad-871e-0e0746ae4283.png)


But on the first collection This corner never is labelled well

**Comment by miguelriemoliveira** [comment]
With this annotation

![image](https://user-images.githubusercontent.com/11350998/164680857-e5392df3-4382-4b3e-90c2-60b77e499a4a.png)

![image](https://user-images.githubusercontent.com/11350998/164680941-fc66e3fb-238b-4a1d-80d7-738cceda530c.png)

it works more or less ... I can try to improve it later on ...
